pkg/cli/sql_util.go

```diff
@@ -1134,6 +1145,11 @@ func formatVal(
 		if colType == "FLOAT4" {
 			width = 32
 		}
+		if math.IsInf(t, 1) {
+			return "Infinity"
```

Comment:
my concern is anyone relying on this right now on a major version who wants a quick patch to some other problem but this breaks something else unexpectedly.